src/payment/AlertCodeMessages.test.jsx

```diff
@@ -15,7 +15,9 @@ const mockStore = configureMockStore();
 describe('SingleEnrollmentCodeWarning', () => {
   it('should render with values', () => {
     const component = (
-      <SingleEnrollmentCodeWarning values={{ courseAboutUrl: 'http://edx.org' }} />
+      <IntlProvider locale="en">
```

Comment:
Surprised adding IntlProvider here was necessary... did it improve the snapshot to do so?

Reply:
Adding it eliminated warnings in the console when running the test.
```
console.error node_modules/react-intl/lib/index.js:1533
    [React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry. Using default message as fallback.
```

Reply:
👍